portfolio/pom.xml

```diff
@@ -35,7 +35,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
```

Comment:
Remember to remove the TODO comment after you've done the thing :)

Reply:
Thanks! I took it out